src/WebApp/Controllers/CustomerManagementController.cs

```diff
@@ -16,14 +19,18 @@ public CustomerManagementController(ICustomerManagementAPI customerManagementAPI
     [HttpGet]
     public async Task<IActionResult> Index()
     {
-        return await _resiliencyHelper.ExecuteResilient(async () =>
+        using (var activity = _activitySource.StartActivity("YourAction"))
+        {
+            activity?.SetTag("your-tag", "value");
+            return await _resiliencyHelper.ExecuteResilient(async () =>
```

Comment:
dit ziet eruit als template code. moet your-tag, value niet iets concreters worden?

Reply:
Ja aangezien het dat wss ook is zal ik het weg doen maar ik heb het zelf eerlijk gezegd niet toegevoegd ofzo.